app/controllers/bikes/recovery_controller.rb

```diff
@@ -24,8 +24,14 @@ def update
     private
 
     def permitted_params
-      params.require(:stolen_record).permit(:date_recovered, :timezone, :recovered_description,
-                                            :index_helped_recovery, :can_share_recovery)
+      params.require(:stolen_record).permit(
+        :date_recovered,
+        :timezone,
+        :recovering_user_id,
```

Comment:
I'd prefer if `recovering_user_id` was taken automatically in the controller, it could even be merged in here `merge(recovering_user_id: current_user&.id)` instead of put in a form in the view.

I don't _think_ anyone will be malicious about this, but why not protect from it just in case.

Reply:
Good call — updated in 0fe013e